src/createRoom.ts

```diff
@@ -268,6 +268,9 @@ export default async function createRoom(opts: IOpts): Promise<string | null> {
                 metricsTrigger: "Created",
             });
         }
+        if (opts.roomType) {
```

Comment:
I think we should check if the `roomType` is a space?

Reply:
also I question why the view_room dispatch immediately above isn't working?

Reply:
This params is passed only once when it is a space

Reply:
But it could also be a different room type in the future

Reply:
![image](https://user-images.githubusercontent.com/67551927/159082559-c88964b2-c6f8-4cd5-b6a4-e40871ead61a.png)

Reply:
This should definitely verify that the type is actually `RoomType.Space` - see e.g. https://github.com/matrix-org/matrix-js-sdk/pull/2249 which adds another possible room type